astroquery/cadc/core.py

```diff
@@ -89,7 +91,11 @@ def __init__(self, url=None, tap_plus_handler=None, verbose=None):
             a url to use instead of the default
         tap_plus_handler : deprecated
         verbose : deprecated
-
+        session: `requests.Session` type
```

Comment:
```suggestion
        auth_session: `requests.Session` type
```

Reply:
How do you do this? I would like to update type too to include `pyvo.auth.authsession.AuthSession` without pushing from local repo.

Reply:
Done. Thanks